src/main/resources/migrations.xml

```diff
@@ -2719,4 +2719,10 @@
 		<!-- Rolling back is not useful -->
 	</changeSet>
 
+	<changeSet id="add-sensitive-attendee-to-ReportPeople" author="jose-west">
```

Comment:
I'd prefer `add-sensitive-attendee-to-reportPeople` (lowercase 'r') since that is the name of the table.